### ISSUE TITLE
Panner

### ISSUE THREAD

**Issue by PandaxCSGO** [discohook/discohook#10]
Is it possible to add a planner? kind that we configure so that it is sent at a specific date & time?

**Comment by jaylineko** [comment]
Yes, but this will be very low priority for now and you'll likely have to keep the tab open for it to send at the desired time.

**Comment by blinkor** [comment]
A planner is an interesting idea!

**Comment by wolfier74** [comment]
Good idea

**Comment by potatoepotato** [comment]
yes
